llama_index/indices/multi_modal/retriever.py

```diff
@@ -52,7 +51,7 @@ def __init__(
         # separate image vector store for image retrieval
         self._image_vector_store = self._index.image_vector_store
 
-        assert isinstance(self._index.image_embed_model, MultiModalEmbedding)
+        # assert isinstance(self._index.image_embed_model, MultiModalEmbedding)
```

Comment:
why comment out this ?

Reply:
oh this was stopping execution because the assertion fails when I replace the `image_embed_model` with the text embed model... Not sure if that's expected behaviour and I didn't investigate yet.

Reply:
Alright I did some digging. We're going to have to relax this assertion statement to assert on `BaseEmbedding` for now since in the event that we want to use the `text_embedding` (which overrides the `embedding` field) for an `ImageNode`, then we must also override the `image_embed_model` to be that which was used to embed the text.